src/components/Pressable/utils.ts

```diff
@@ -119,12 +119,72 @@ const adaptTouchEvent = (event: GestureTouchEvent): PressableEvent => {
   };
 };
 
+type StylePropKeys = (keyof ViewStyle)[];
+
+// Source:
+// - From ViewStyle extracted FlexStyle sub-interface which contains all of the box-model manipulating props.
+// - From FlexStyle handpicked those styles, which act on the inner part of the box-model.
+const innerStyleKeys = [
+  'alignContent',
+  'alignItems',
+  'flexBasis',
+  'flexDirection',
+  'flexWrap',
+  'rowGap',
+  'gap',
+  'columnGap',
+  'justifyContent',
+  'overflow',
+  'padding',
+  'paddingBottom',
+  'paddingEnd',
+  'paddingHorizontal',
+  'paddingLeft',
+  'paddingRight',
+  'paddingStart',
+  'paddingTop',
+  'paddingVertical',
+  'start',
+  'end',
+  'direction', // iOS only
+] as StylePropKeys;
+
+const extractStyles = (from: ViewStyle, keys: StylePropKeys): ViewStyle => {
+  if (!from) {
+    return {} as ViewStyle;
+  }
+
+  return Object.fromEntries(
+    keys
+      .map((key) => [key, from[key]])
+      .filter(([_key, value]) => value !== undefined)
```

Comment:
```suggestion
    keys
      .filter((key) => key in from)
      .map((key) => [key, from[key]])
```

I am not sure about whether the `in` syntax is better, but the order should be swapped.

Reply:
swapped the order.